lib/wrappers/daoCreator.ts

```diff
@@ -99,7 +98,18 @@ export class DaoCreatorWrapper extends ContractWrapperBase {
       controllerAddress = 0;
     }
 
-    const totalGas = computeForgeOrgGasLimit(options.founders.length);
+    const paramsArray = [
+      options.name,
+      options.tokenName,
+      options.tokenSymbol,
+      options.founders.map((founder: FounderConfig) => web3.toBigNumber(founder.address)),
+      options.founders.map((founder: FounderConfig) => web3.toBigNumber(founder.tokens)),
+      options.founders.map((founder: FounderConfig) => web3.toBigNumber(founder.reputation)),
+      controllerAddress,
+      options.tokenCap];
+
+    // always use estimateGas or will fail when using non-universal controller
```

Comment:
"or will fail when using non-universal controller ? why ?

Reply:
It runs out of gas.  I don't know why the default gas limit doesn't work.